### DISCUSSION TITLE
Impossibile utilizzare DUCKDNS dopo nuova installazione HA

### DISCUSSION THREAD

**Discussion by magomer** [indomus/forum#1910] [HUB: Home Assistant]
Ciao a tutti.
E' da poco più di 1 anno che smanetto con HA. Utilizzavo una VM su windows proprio per smanettare.
Tutto ok anche dall'esterno.
Adesso ho fatto una macchina (PC portatile) con HA OS seguendo la guida INDOMUS. Quello che non riesco più a far funzionare è il servizio DUCKDNS.
Avviando il servizio la configurazione dello stesso è OK, visualizzo la creazione dei certificati e l'ultimo messaggio e done.
Apro la porta su router (già fatto altro volte quindi dovrei essere capace), in locale mi fa entrare in HTTPS (giustamente aggiungo).
Ma dall'esterno niente da fare. 
Prima di scrive questo messaggio ho aspettato una settimana quindi non penso che il problema sia DUCKDNS...
Avete qualche suggerimento ?
Grazie 

P.S. : NON funziona più neanche la vecchia macchina

**Comment by snakuzzo** [comment]
> Avviando il servizio la configurazione dello stesso è OK, visualizzo la creazione dei certificati e l'ultimo messaggio e done.

Cosa utilizzi per la gestione dei certificati ?

> Apro la porta su router (già fatto altro volte quindi dovrei essere capace), in locale mi fa entrare in HTTPS (giustamente aggiungo).

Visto che dici che in locale ti risponde correttamente in https, immagino tu non stia utilizzando un reverse proxy (ex.Caddy, NGINX, etc)

> Ma dall'esterno niente da fare.
Prima di scrive questo messaggio ho aspettato una settimana quindi non penso che il problema sia DUCKDNS...

A cosa servirebbe "aspettare una settimana" ? Se funziona, funziona da subito, non c'è bisogno di aspettare niente.
Comunque, premesso che il FQDN sia associato all'IP pubblico giusto, se in LAN ti funziona (es. `https://192.168.1.10:8123`) e dall'esterno no (es. `https://tuodominio.duckdns.org`), unica cosa che mi viene in mente è la regola di port forwarding errata. Sicuro che la VM (o l'host...a seconda di come hai configurato la network) abbia l'ip giusto ?

>  P.S. : NON funziona più neanche la vecchia macchina

Stesso discorso di cui sopra

**Reply by magomer** [reply]
Prima di tutto GRAZIE dell'interessamento.
Allora parto dalla più semplice, ho atteso dei gg visto che ogni tanto DUCK ha problemi.
Eliminando come possibile problematica la gestione dell' IP, visto che un pò me ne intendo e che, anche se così non fosse, sulla vecchia macchina NON ho modificato niente, rimane la gestione dei certificati che ho dato per scontato fosse OK visto l'assenza di messaggi di errore.
Hai qualche link  per leggere come verificare e rinnovarli ?
Puoi suggerirmi qualcosa ?
Grazie

Marco

**Reply by Scialla** [reply]
> Comunque, premesso che il FQDN sia associato all'IP pubblico giusto

Io non sarei così certo della premessa..

Verifica che l'IP sul sito di duckdns sia uguale all'IP pubblico che ti è stato assegnato dal provider..

**Reply by magomer** [reply]
Allora ho rifatto tutto.
Nuova installazione di HA SO.
Configurato solo File editoe e Duckdns.
IP Pubblico verifica e corrisponde a quello di duck.
Aperta porta 8123.
In locale mi collego sia con IP di HA che con indizzo duck :8123 sempre con HTTPS.
Non ho idea di come verifica e rinnovare i certificati.
Hai suggerimenti anche qualche linlk da leggere ?
Grazie

Marco

PS Per IP ho anche spento il router per farlo cambiare e verifica se Duck cambiava. Verifica positiva

**Comment by magomer** [comment]
Aggiungo un'informazione.
Inserito sensore scadenza certificato, mi da ancora 3 mesi.....
MI STO QUASI ARRENDENDO ..........
